test/test_quantized.py

```diff
@@ -82,6 +82,7 @@ def pool_output_shape(input_size, kernel_size, padding, stride,
         output_size += 1
     return output_size
 
+
```

Comment:
nit: Extra line

Reply:
Oh, I think this line has to be here -- isn't it python style to have two blank lines before class defs?